tests/by-util/test_ls.rs

```diff
@@ -4298,6 +4297,115 @@ fn test_ls_context_format() {
     }
 }
 
+/// Helper function to validate `SELinux` context format
+/// Returns the context parts if valid, panics with descriptive message if invalid
+#[cfg(feature = "feat_selinux")]
+fn validate_selinux_context(context: &str) {
+    assert!(
+        context.contains(':'),
+        "Expected SELinux context format (user:role:type:level), got: {}",
+        context
+    );
+
+    let context_parts: Vec<&str> = context.split(':').collect();
+    assert_eq!(
+        context_parts.len(),
```

Comment:
You can simplify it by using `count`:
```suggestion
    assert_eq!(
        context.split(':').count(),
```